app/experimenter/experiments/forms.py

```diff
@@ -1271,10 +1272,42 @@ class NormandyIdForm(ChangeLogMixin, forms.ModelForm):
     normandy_id = forms.IntegerField(
         label="Recipe ID",
         widget=forms.TextInput(
-            attrs={"class": "form-control", "placeholder": "Recipe ID"}
+            attrs={"class": "form-control", "placeholder": "Main Recipe ID"}
         ),
     )
 
+    other_normandy_ids = forms.CharField(
+        label="Other Recipe IDs",
+        widget=forms.TextInput(
+            attrs={
+                "class": "form-control",
+                "placeholder": "Other Recipe IDs (Optional)",
+            }
+        ),
+        required=False,
+    )
+
+    def clean_other_normandy_ids(self):
+        other_ids_list = []
+
+        if self.cleaned_data["other_normandy_ids"]:
+            ids = re.split(",", self.cleaned_data["other_normandy_ids"])
+            for id in ids:
+                if re.match(r"^[0-9]+$", id.strip()):
+                    if "normandy_id" in self.cleaned_data:
```

Comment:
This will have to be here because the field is required and that will get checked before this validation happens.

Reply:
Actually you can't check the cleaned value of other fields from a specific fields clean method, you have to do that in a `clean` method for the whole form because fields can be validated in any order.